### DISCUSSION TITLE
MultipleChoice as a question component.

### DISCUSSION THREAD

**Discussion by n-mitchell** [ditdot-dev/vue-flow-form#176] [Q&A] [answered]
Hi,

Great work on this.

I'm looking for the correct format to specify the options when using MultipleChoice as a question component?

` <question
        type="MultipleChoice"
        title="Are we looking at this policy just for yourself, or for you and your partner?"
        v-model="policyfor"
      >
      </question>`

Where should I be putting the options?

**Comment by EkaterinaVu** [comment, marked as answer]
@n-mitchell ,  the correct format to specify the MultipleChoice options is:

` <question type="multiplechoice" title="Multiple choice question" :options="[{label: 'Answer 1', value: 'answer_1'}, {label: 'Answer 2', value: 'answer_2'}]"></question>`
